### DISCUSSION TITLE
Align text about blog

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#1405] [Q&A]
I am trying to create a Blog. However, in the about.md the content is aligned to the centre, while I want it to be aligned on the left as when I write a post. 

I cannot find the relevant information. Can you please help

**Comment by mcanouil** [comment]
You can change the template as described in the documentation: https://quarto.org/docs/websites/website-about.html#templates

Or you can use fenced div and CSS to customised your document.

**Reply by ghost** [reply]
Got it on the first, but I am less experienced or unfamiliar with the latter. Any example, somewhere in the documentation?

**Reply by mcanouil** [reply]
An example of fenced div to center columns in reveal.js slides can be found on #1386 discussion.
In the documentation:
- https://quarto.org/docs/authoring/markdown-basics.html#divs-and-spans
- https://quarto.org/docs/authoring/markdown-basics.html#other-blocks

**Comment by ghost** [comment]
I fixed my problem by just adding   `<div align="left">`

For example: 

```
<div align="left">
Welcome to my website! 

```
